ibu-imager/internal/seedcreator/seedcreator.go

```diff
@@ -32,21 +32,19 @@ type SeedCreator struct {
 	backupDir         string
 	kubeconfig        string
 	containerRegistry string
-	backupTag         string
 	authFile          string
 }
 
 // NewSeedCreator is a constructor function for SeedCreator
 func NewSeedCreator(log *logrus.Logger, ops ops.Ops, ostreeClient *ostree.Client, backupDir,
-	kubeconfig, containerRegistry, backupTag, authFile string) *SeedCreator {
+	kubeconfig, containerRegistry, authFile string) *SeedCreator {
 	return &SeedCreator{
 		log:               log,
 		ops:               ops,
 		ostreeClient:      ostreeClient,
 		backupDir:         backupDir,
 		kubeconfig:        kubeconfig,
 		containerRegistry: containerRegistry,
```

Comment:
Can we rename this field and argument as well? ie. --image instead of --registry? Or maybe add --image and (temporarily) keep --registry so dev can use both for the meantime?

Reply:
I would directly rename it, since we're so early in the process, and that would force people also to include the tag .... delaying it will only make the same trauma of changing in a while :D

WDYT?

Reply:
I'm completely fine with that!

Reply:
Done :)